parametric/test_headers_tracecontext.py

```diff
@@ -109,7 +109,7 @@ def test_traceparent_header_name(test_agent, test_library):
 @temporary_enable_optin_tracecontext()
 @pytest.mark.skip_library("dotnet", "Bug: Header search is currently case-sensitive")
 @pytest.mark.skip_library("golang", "not implemented")
-@pytest.mark.skip_library("nodejs", "not implemented")
+@pytest.mark.skip_library("nodejs", "nodejs lower-cases all header names")
```

Comment:
Would the test pass if we didn't skip it? If so, I'd like to run the test to demonstrate that these inputs have the correct output and we could leave a code comment about the nodejs behavior if you'd like

Reply:
It does not because we don't lowercase it _ourselves_ and these "headers" passed through RPC are not _real_ headers so because of the test only _immitating_ HTTP we don't actually get the test headers in lowercase even though that's impossible in _real_ HTTP.

Reply:
I made a change to the RPC service to simulate the lowercasing.